docs/source/en/model_doc/dpt.mdx

```diff
@@ -33,7 +33,8 @@ This model was contributed by [nielsr](https://huggingface.co/nielsr). The origi
 A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with DPT.
 
 - Demo notebooks for [`DPTForDepthEstimation`] can be found [here](https://github.com/NielsRogge/Transformers-Tutorials/tree/master/DPT).
-- See also: [Semantic segmentation task guide](./tasks/semantic_segmentation)
+- [Semantic segmentation task guide](../tasks/semantic_segmentation)
+- [Monocular depth estimation task guide](../tasks/monocular_depth_estimation.mdx)
```

Comment:
@MKhalusova - I've just noticed that these links shouldn't have the `.mdx` at the end. Could you open another PR to remove them?

Reply:
Thanks for adding! Noticed images aren't rendered properly, might need to double check those links as well

Reply:
Update, have merged the PR on the hub, that's working now 👍

Reply:
Thank you @NielsRogge !